RuleDocumentation/ProvideCommentHelp.md

```diff
@@ -4,9 +4,11 @@
 
 ## Description
 
-Comment based help should be provided for all PowerShell commands. This test only checks for the presence of comment based help and not on the validity or format.
+Comment based help should be provided for all PowerShell commands. This test only checks for the
+presence of comment based help and not on the validity or format.
 
-For assistance on comment based help, use the command ```Get-Help about_comment_based_help``` or the article, "How to Write Cmdlet Help" (https://go.microsoft.com/fwlink/?LinkID=123415).
+For assistance on comment based help, use the command `Get-Help about_comment_based_help` or the
+article, [How to Write Cmdlet Help](https://go.microsoft.com/fwlink/?LinkID=123415).
```

Comment:
All the links on this link are broken, also seems like it is a super outdated (made 2007) doc, not sure if there is a more up to date one about writing cmdlet helps

Reply:
Yep, I will fix the links to newer content.

Reply:
Fixed